packages/ag-charts-community/src/chart/chart.ts

```diff
@@ -1886,6 +1893,17 @@ export abstract class Chart extends Observable {
 
         debug(`Chart.applyAxes() - creating new axes instances; seriesStatus: ${seriesStatus}`);
         chart.axes = this.createAxis(axes, skip);
+
+        const axisGroups: { [Key in ChartAxisDirection]: Group[] } = {
+            [ChartAxisDirection.X]: [],
+            [ChartAxisDirection.Y]: [],
+        };
+
+        chart.axes.forEach((axis) => axisGroups[axis.direction].push(axis.getAxisGroup()));
+
+        this.ctx.regionManager.updateRegion(REGIONS.HORIZONTAL_AXES, ...axisGroups[ChartAxisDirection.X]);
+        this.ctx.regionManager.updateRegion(REGIONS.VERTICAL_AXES, ...axisGroups[ChartAxisDirection.Y]);
```

Comment:
Side-note: There is a lot of axis-related functionality being added to `Chart`, I think we maybe missing an abstraction around the axes (naively an `AxisGroup` at the least) which could have some of these responsibilities.

Reply:
I'm not persuaded that an `AxisGroup` class is necessary. However, it might make more sense to move this logic into the `CartesianChart` sub-class because the concept of horizontal & vertical axes is not applicable to all series types.

Reply:
tried moving that stuff to cartesian chart but it caused some intergrated charts cross-filtering snapshot failures and it wasn't obvious to me what was causing that so undid it. Leaving as is for now?